olp-cpp-sdk-dataservice-read/tests/VersionedLayerClientOfflineTest.cpp

```diff
@@ -0,0 +1,407 @@
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
```

Comment:
Rename to `VersionedLayerClientTest`. Also, do we want to move some of these tests into integration tests?